pkg/client/kubernetes/clientmap/internal/seed_clientmap.go

```diff
@@ -65,6 +67,10 @@ func (f *SeedClientSetFactory) NewClientSet(ctx context.Context, k clientmap.Cli
 				Scheme: kubernetes.SeedScheme,
 			},
 		),
+		kubernetes.WithUncached(
+			&corev1.Event{},
+			&eventsv1.Event{},
+		),
```

Comment:
I think the unit tests should be adapted to this change.

Reply:
Of course, done.